docs/pages/en/integrations/markdown-it-unlazy-img/index.md

```diff
@@ -48,6 +48,14 @@ import { defineConfigWithTheme } from 'vitepress'
 import { UnlazyImages } from '@nolebase/markdown-it-unlazy-img' // [!code ++]
 
 export default defineConfigWithTheme({
+    vue: {
+    template: {
+      transformAssetUrls: {
+        // Other configurations...
```

Comment:
I remember this option is required to specify all the tags and elements since they dont support merging the options.

Reply:
> I remember this option is required to specify all the tags and elements since they dont support merging the options.

That's what I thought at first, but I tested it locally by specifying only the `NolebaseUnlazyImg` tag, and it works just fine.🤔